generators/generator-base.js

```diff
@@ -2027,6 +2027,88 @@ module.exports = class extends PrivateBase {
         return jhipsterUtils.getAllJhipsterConfig(generator, force, configRootPath);
     }
 
+    migrateAllBlueprints() {
+        this.jhipsterConfig = this.jhipsterConfig || this._getStorage('generator-jhipster');
+        this.jhipsterOldVersion = this.jhipsterOldVersion || this.jhipsterConfig.get('jhipsterVersion');
```

Comment:
With 6.6.0 and app blueprint, jhipsterVersion can be on blueprint config. Use getAllJhipsterConfig at this.

Reply:
Quit if old jhipster is greater than 6.6.0